apps/vite/src/components/calendar/Today.tsx

```
@@ -1,55 +1,120 @@
+              />
+              <p className="text-sm text-gray-600">
+                सूर्यास्त: {data.panchangaDetails?.times.sunset}
+              </p>
+            </div>
+            <div className="flex justify-end">
+              <button
+                className="text-sm rounded-lg flex-1 items-end text-end transition-all duration-300 text-indigo-500 hover:text-indigo-600"
+                onClick={() => setIsOpen(!isOpen)}
+              >
+                आजको पंचांग{" "}
+                {isOpen ? (
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)